app/templates/404.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+{% block body %}
+<main class="done-content">
+    <p><i class="done-icon fa fa-thumbs-down"></i></p>
+    <h1>Data8 Interact Server</h1>
+    <p>Looks like your request was malformed. It must contain either the file
+        param <br>
+        <a href="http://localhost:8002/?file=http://localhost:8000/test.ipynb">
+            <code>http://localhost:8002/?file=http://localhost:8000/test.ipynb</code>
+        </a>
+        or both the repo and path params. <br>
+        <a href="http://localhost:8002/?repo=data8assets&path=labs/lab01">
+            <code>http://localhost:8002/?repo=data8assets&path=labs/lab01</code>
```

Comment:
It looks like we'll show these URLs with localhost in production too. Think we can just change them to data8.b.e? I think it's better to show production URLs in development rather than the other way around since students will be less confused.

Maybe we can add a note telling the student that this probably isn't their fault and they should contact their instructor.